src/components/ServiceSearch.jsx

```diff
@@ -102,32 +95,117 @@ const ServiceSearch = () => {
           className={classes.searchBar}
           style={styles.searchBar}
         >
-          <FormControl>
-            <OutlinedInput
-              data-cy="search-query"
-              onChange={(e) => setSearchQuery(e.target.value)}
-              color="secondary"
-              value={searchQuery}
-              placeholder={"Search for a community service..."}
-              aria-describedby="Search for self care services"
-              className={classes.queryInput}
-              style={styles.queryInput}
-            />
-            <FormHelperText style={styles.helperText}>
-              Try "befriending" or "sports".
-            </FormHelperText>
-          </FormControl>
-          <Button
-            data-cy="search-submit"
-            onClick={() => performSearch()}
-            variant="contained"
-            color="secondary"
-            className={classes.searchButton}
-            style={styles.searchButton}
-            disableElevation
-          >
-            <SearchIcon />
-          </Button>
+          <Grid container justify={mobile ? "flex-start" : "center"} style={{flexWrap: "wrap-reverse"}}>
```

Comment:
theres a lot of different style types here now. Wouldn’t mind grouping it up a bit?

Reply:
I agree, but for some reason only justify attribute actually works, if I do it in style it has no effect.

Reply:
Hmm yeah i think iv'e been there as well. but i think it is possible to fix but if it works now its okay so we can keep going :) and put it up as a chore for the future